packages/gravity-ui-web/src/ui-lib/sass/01-tools/_container.scss

```diff
@@ -1,6 +1,6 @@
 // a simple mixin to create a container for the page
 @mixin grav-l-container($no-margin: false) {
-  width: 100%;
+  width: calc(100% - #{2 * $grav-page-content-inset});
```

Comment:
Should this be in the `@if (not $no-margin)` below?

Reply:
Yes, you're right. Will push a fix.

Although, TBH, I don't think that option is used anywhere.